src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -308,6 +308,9 @@ void VM_Version::get_processor_features() {
       warning("UseAESIntrinsics enabled, but UseAES not, enabling");
       UseAES = true;
     }
+    if (FLAG_IS_DEFAULT(UseAESCTRIntrinsics)) {
+      FLAG_SET_DEFAULT(UseAESCTRIntrinsics, false);
+    }
```

Comment:
`UseAESCTRIntrinsics` is already `false` by default, so this seems to be excessive. No harm doing this anyway.

Reply:
+1